src/links/v1/providers.json

```diff
@@ -78,10 +78,10 @@
       "type": "links",
       "id": "mp",
       "attributes": {
-        "name": "materialsproject.org",
-        "description": "",
-        "base_url": null,
-        "homepage": null,
+        "name": "The Materials Project",
+        "description": "The Materials Project OPTIMADE endpoint",
```

Comment:
Or maybe my comment above is more apt for _this_ description, since this is what will describe the Materials Project from the list of providers.